app/routes/admin/content/index.js

```diff
@@ -8,5 +8,10 @@ export default Route.extend({
   },
   model() {
     return this.get('store').queryRecord('setting', {});
+  },
+  actions: {
+    willTransition() {
+      this.get('currentModel').rollbackAttributes();
```

Comment:
Mmm ... actually ... keep your previous one. `controller.model`. `currentModel` is a private property and could change in future.

Ref: https://github.com/emberjs/ember.js/issues/11760